### DISCUSSION TITLE
custom rewrite settings

### DISCUSSION THREAD

**Discussion by MaxPowers1337** [nginx-proxy/nginx-proxy#2209] [Q&A]
Hello, 

i've a vhost which is reachable via Path-based Routing:

VIRTUAL_PATH=/myapp

I now need to set the following nginx-proxy-setting for the application to work 

```
location /myapp/ {
    [...]
    rewrite ^/myapp/(.*) /$1 break;
}
```

Is there a possibility for this? Is this perhaps the VIRTUAL_DEST setting, which i havent understand yet? :)

**Comment by buchdag** [comment]
@MaxPowers1337 did you find the answer to your question ?

**Reply by MaxPowers1337** [reply]
When writing this post i was forced to think about the problem in order to phrase my problem. And by clicking on 'create'  .. i noticed that in fact i need [Per Virtual Path Location Configuration](https://github.com/nginx-proxy/nginx-proxy#per-virtual_path-location-configuration) for adding the rewrite within the location block :)

**Reply by buchdag** [reply]
The power of rubberducking 👍
